src/pythermalcomfort/utilities.py

```diff
@@ -529,3 +536,1156 @@ def mapping(value, map_dictionary, right=True):
     "Cotton vacuum metallized with aluminium": 0.05,
     "Glass fiber vacuum metallized with aluminium": 0.07,
 }
+
+# This dictionary contains the local and the whole body clothing insulation of typical clothing ensemble.
+# It is based on the study by Juyoun et al. (https://escholarship.org/uc/item/18f0r375)
+# and by Nomoto et al. (https://doi.org/10.1002/2475-8876.12124)
+# Please note that value for the neck is the same as the measured value for the head
+# and it does not take into account the insulation effect of the hair.
+# Typically, the clothing insulation for the hair are quantified by assuming a head covering of approximately 0.6 to 1.0 clo.
+local_clo_typical_ensembles =  {
```

Comment:
these dictionaries are also only specific to the JOS3 model so we may want to move them to the `jos3_funcitons` folder.